ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml

```diff
@@ -44,15 +47,22 @@ tests:
     make lint
   container:
     clone: true
-    from: golangci-lint-shellcheck
+    from: test-bin
+- as: helm-lint
+  commands: |
+    export XDG_CACHE_HOME=/tmp/.cache
```

Comment:
This was needed for `golangci-lint`. Is it also needed for `helm lint`?

Reply:
Good question, checking

Reply:
```
$XDG_CACHE_HOME defines the base directory relative to which user-specific non-essential data files should be stored. If $XDG_CACHE_HOME is either not set or empty, a default equal to $HOME/.cache should be used.
```

It doesn't seems related specifically for golangci-lint so I guess there is no harm to have it here but I don't really know.

Reply:
I am wondering if `helm lint` writes anything into the cache directory at all. Could we try removing that line?

Reply:
Removed.